scripts/dashboard/install-cephadm-e2e-deps.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -ex
+
+on_error() {
+    if [ "$1" != "0" ]; then
+        printf "\n\nERROR $1 thrown on line $2\n\n"
+        printf "\n\nCollecting info...\n\n"
+        sudo journalctl --since "10 min ago" --no-tail --no-pager -x
+        printf "\n\nERROR: displaying podman logs:\n\n"
+        podman logs -l
+        printf "\n\nTEST FAILED.\n\n"
+    fi
+}
+
+trap 'on_error $? $LINENO' ERR
+
+sudo apt -y install libvirt-daemon-system libvirt-daemon-driver-qemu qemu-kvm libvirt-clients
+
+sudo usermod -aG libvirt $(id -un)
+newgrp libvirt  # Avoid having to log out and log in for group addition to take effect.
+sudo systemctl enable --now libvirtd
+
+# Add podman repo for ubuntu focal.
+VERSION_ID='20.04'
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_${VERSION_ID}/Release.key -O- | sudo apt-key add -
+sudo apt update -y
+
+# Install required deps.
+sudo apt install -y nodejs npm openssh-server podman
+
+mkdir -p ~/.kcli ~/.ssh
+if [[ ! -f "$HOME/.ssh/id_rsa" ]]; then
+    ssh-keygen -t rsa -q -f "$HOME/.ssh/id_rsa" -N ""
+fi
+sudo ls -la ~/.ssh
+
+: ${KCLI_CONTAINER_IMAGE:='docker.io/jolmomar/kcli'}
+
+echo "#!/usr/bin/env bash
+
+sudo podman run --net host --security-opt label=disable \
+    -v ${HOME}/.ssh:/root/.ssh \
+    -v ${HOME}/.kcli:/root/.kcli \
+    -v ${PWD}:/workdir \
+    -v /var/lib/libvirt/images:/var/lib/libvirt/images \
+    -v /var/run/libvirt:/var/run/libvirt \
+    -v /var/tmp:/ignitiondir \
+    ${KCLI_CONTAINER_IMAGE} \""'${@}'"\"
+" | sudo tee /usr/local/bin/kcli
+sudo chmod +x /usr/local/bin/kcli
+
+podman info
+sudo podman container prune -f
+
+kcli create pool -p /var/lib/libvirt/images default
```

Comment:
Does this ever get cleaned up?

Reply:
I do some cleanup in the associated ceph script but it does not include this.

Following your idea of having dedicated builders with libvirt+podman(or docker) and taking into account that the VM image is never modified, not deleting the image can improve the speed of the job.
Does it make sense to not do the cleanup in the dedicated nodes?

Reply:
@djgalloway As per our last conversation we agreed on using the `node: huge && focal && x86_64` machines and a dedicated libvirt pool and network (ceph-dashboard) to avoid conflicts in the future.